app/controllers/mail_subscriptions_controller.rb

```diff
@@ -9,6 +9,8 @@ class MailSubscriptionsController < ApplicationController
   before_action :set_user
   before_action :set_type
 
+  protect_from_forgery
```

Comment:
What is this change for? `ApplicationController` already sets `protect_from_forgery with: :exception`

Reply:
The POST request with `List-Unsubscribe=One-Click` made by the email client should not throw an exception. 

`protect_from_forgery` without `with` defaults to `with: : null_session`. I have now changed this to `skip_forgery_protection` to make it more explicit that we don't care about CSRF here.

Reply:
Ah, I did not know the email client itself would do a POST request. That makes sense, although no session should be needed, so `:null_session` should be fine.

Reply:
Ok, reverted to `protect_from_forgery with: :null_session` :-)